targets/docker/README.md

```diff
@@ -0,0 +1,37 @@
+# `Dockerfile`
+
+# run build
+
+Run from the top-level directory:
+
+```sh
+jerryscript $ ls -ah
```

Comment:
Any reason why you want to list the contents of the JerryScript top-level directory here?